awswrangler/_executor.py

```diff
@@ -38,6 +38,7 @@ def __init__(self, use_threads: Union[bool, int]):
         self._exec: Optional[concurrent.futures.ThreadPoolExecutor] = None
         self._cpus: int = _utils.ensure_cpu_count(use_threads=use_threads)
         if self._cpus > 1:
+            _logger.debug("Initializing ThreadPoolExecutor with %d workers", self._cpus)
```

Comment:
Wyt about moving this into the `ThreadPoolExecutor` itself?

Reply:
What do you mean? `ThreadPoolExecutor` is a build-in construct in Python

Reply:
Ah, apologies this is already in `awswrangler._executor._ThreadPoolExecutor`, I thought it was outside :)